### ISSUE TITLE
Stencil ignores configuration for specific files when building for production

### ISSUE THREAD

**Issue by siedentm** [ionic-team/stencil#190]
**Stencil version:**

```
@stencil/core@0.0.6-2
```

**I'm submitting a ...**  (check one with "x")
[ x ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**
The compiler ignores stencil.config.js settings when building without --dev (`npm run build`).
This counts for minification and hash names for the loader, core and corePolyfill. Component bundles are getting built as intended.

**Related code:**
stencil.config.js:
```
exports.config = {
  namespace: 'mybundle',
  generateDistribution: true,
  generateWWW: false,
  hashFileNames: false,
  serviceWorker: false,
  minifyCss: false,
  minifyJs: false,
  bundles: [
    { components: ['my-name'],
      components: ['my-name2'] }
  ]
};
```

**Comment by jgw96** [comment]
Just tested this and can confirm this issue. Thanks for opening this issue!

**Comment by adamdbradley** [comment]
Should be fixed as of `0.0.6-10`. Thanks!
